sdk/core/azure-core/src/main/java/com/azure/core/http/HttpHeader.java

```diff
@@ -23,4 +25,8 @@
     public HttpHeader(String name, String value) {
         super(name, value);
     }
+
+    public HttpHeader(String name, List<String> values) {
```

Comment:
nit: Javadocs

Reply:
JavaDoc is coming through now as I get towards the end of the investigation.